sdk/src/Core/Amazon.Runtime/Telemetry/DefaultTelemetryProvider.cs

```diff
@@ -0,0 +1,71 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+using Amazon.Runtime.Telemetry.Metrics;
+using Amazon.Runtime.Telemetry.Tracing;
+
+namespace Amazon.Runtime.Telemetry
+{
+    /// <summary>
+    /// Default implementation of <see cref="TelemetryProvider"/> that uses no-op providers by default.
+    /// </summary>
+    public class DefaultTelemetryProvider : TelemetryProvider
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DefaultTelemetryProvider"/> class with no-op providers.
+        /// This setup prevents any telemetry actions from being performed unless explicitly registered using the
+        /// registration methods.
+        /// </summary>
+        public DefaultTelemetryProvider()
+        {
+            TracerProvider = new NoOpTracerProvider();
```

Comment:
After we add `System.Diagnostics`, Will add a condition here to use `NoOpTracerProvider` only for .NET framework 3.5 and 4.5, and use the `System.Diagnostics` implementation for .net standard and above

Reply:
What do you mean add `System.Diagnostics`? That is part of the base class library in .NET Standard/Core and nothing needs to be added.

Reply:
I mean the adapter that we will create to wrap `System.Diagnostics` instead of the NoOp, it should be something like this 

```
#if BCL
    TracerProvider = new NoOpTracerProvider();
    MeterProvider = new NoOpMeterProvider();
#else
   
    TracerProvider = new DiagnosticsTracerAdapter();
    MeterProvider = new DiagnosticsMeterAdapter();
#endif
```